src/main/asciidoc/_chapters/getting_started.adoc

```diff
@@ -55,80 +55,34 @@ See <<java,Java>> for information about supported JDK versions.
 . Choose a download site from this list of link:https://www.apache.org/dyn/closer.lua/hbase/[Apache Download Mirrors].
   Click on the suggested top link.
   This will take you to a mirror of _HBase Releases_.
-  Click on the folder named _stable_ and then download the binary file that looks like
-  _hbase-<version>-bin.tar.gz_.
+  Click on the folder named _stable_ and then download the binary file that ends in _.tar.gz_ to your local filesystem.
+  Do not download the file ending in _src.tar.gz_ for now.
 
-. Extract the downloaded file and change to the newly-created directory.
+. Extract the downloaded file, and change to the newly-created directory.
 +
+[source,subs="attributes"]
 ----
-$ tar xzvf hbase-<version>-bin.tar.gz
-$ cd hbase-<version>/
+
+$ tar xzvf hbase-{Version}-bin.tar.gz
+$ cd hbase-{Version}/
 ----
 
-. Set the `JAVA_HOME` environment variable in _conf/hbase-env.sh_.
-  First, locate the installation of `java` on your machine. On Unix systems, you can use the
-  _whereis java_ command. Once you have the location, edit _conf/hbase-env.sh_ file, found inside
-  the extracted _hbase-<version>_ directory, uncomment the line starting with `#export JAVA_HOME=`,
-  and then set it to your Java installation path.
+. You must set the `JAVA_HOME` environment variable before starting HBase.
+  To make this easier, HBase lets you set it within the _conf/hbase-env.sh_ file. You must locate where Java is
+  installed on your machine, and one way to find this is by using the _whereis java_ command. Once you have the location,
+  edit the _conf/hbase-env.sh_ file and uncomment the line starting with _#export JAVA_HOME=_, and then set it to your Java installation path.
 +
-.Example extract from _conf/hbase-env.sh_ where `JAVA_HOME` is set
+.Example extract from _hbase-env.sh_ where _JAVA_HOME_ is set
```

Comment:
hbase-env.sh, unexpectedly, lives in conf dir so this change is incorrect (hbase-env.sh manufactures config....)

Reply:
This is just reverting the book to as it was before HBASE-24106.

What's wrong with hbase-env.sh being in `conf`?

Reply:
I read this change wrong. Ignore.